x/cheqd/utils/did_test.go

```diff
@@ -10,8 +10,8 @@ func TestIsDid(t *testing.T) {
 		valid bool
 		did   string
 	}{
-		{true, "did:cheqd:test:wyywywywyw"},
-		{true, "did:cheqd:test:wyywywywyw:sdadasda"},
+		{true,  "did:cheqd:test:000000wyywywywyw"},
```

Comment:
This is a great example of something that includes no special characters, but fails Base58 encoding spec.